File: leetcode/rust/find-words-containing-character.rs
impl Solution {
    pub fn find_words_containing(words: Vec<String>, x: char) -> Vec<i32> {
        let mut result = Vec::new();

        for (idx, word) in words.iter().enumerate() {
            if word.contains(x) {
                result.push(idx as i32);
            }
        }

        result
    }
}

File: leetcode/go/find-words-containing-character.go
package main

import "strings"

func FindWordsContaining(words []string, x byte) []int {
	result := make([]int, 0)

	for i, word := range words {
			if strings.Contains(word, string(x)) {
					result = append(result, i)
			}
	}

	return result
}
